hell/adventcalendar/2025/16/index.md

```diff
@@ -1,35 +1,114 @@
 ---
-title: "Something something about the Invoker Commands API"
-author: "Your Name"
-author_bio: "Your short bio"
+title: "Controlling dialogs and popovers with the Invoker Commands API"
+author: "Aubrey Sambor"
+author_bio: "Aubrey Sambor is a front-end developer and accessibility advocate. She's one of the organizers of [A11yTalks](https://a11ytalks.com), a monthly virtual meetup featuring speakers and conversations around digital accessibility. She loves CSS, fountain pens, knitting, coffee, and sushi."
 date: 2025-12-16
 author_links:
-  - label: "Site"
-    url: "https://linktoyourblog123.com"
-    link_label: "linktoyourblog123.com"
+  - label: "Aubrey's website"
+    url: "https://aubreysambor.com"
+    link_label: "aubreysambor.com"
+  - label: "Aubrey on Mastodon"
+    url: "https://labyrinth.social/@starshaped"
+    link_label: "@starshaped@labyrinth.social"
 intro: "<p>Short introductory text</p>"
 image: "advent25_16"
 ---
-Some text.
-Some text.
+The [Invoker Commands API](https://developer.mozilla.org/en-US/docs/Web/API/Invoker_Commands_API) is a new way to control popovers and modal dialogs without having to write JavaScript. Dialogs have been available in all modern browsers since March 2022, and the Popover API is available in all modern browsers as of January 2025.
```

Comment:
wonder if it's worth introducing Invoker Commands as a way to assign behaviours to buttons, rather than a thing just for popovers and dialogs?